server/models/Task.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose'
+const taskSchema = new mongoose.Schema(
+    {
+        createdAt:Date,
+        updatedAt : Date,
+        title:String,
+        description:String,
+        tasks:[{ type : ObjectId, ref: 'Task' }],
```

Comment:
ok this is good. 
But lets avoid making relations and make it as Array of strings

Reply:
Done